File: Data Structures & Algorithms/two-integer-sum/submission-0.js
class Solution {
    /**
     * @param {number[]} nums
     * @param {number} target
     * @return {number[]}
     */
    twoSum(nums, target) {
        for (let c = 0; c < nums.length; c++) {
            for(let x = c+1; x < nums.length; x++){
                if(target == (nums[c] + nums[x])){
                    return [c,x]
                }
                
            }
        }
    }
}

File: Data Structures & Algorithms/two-integer-sum-ii/submission-0.cpp
class Solution {
public:
    vector<int> twoSum(vector<int>& numbers, int target) {
        vector<int> ans;
        int left = 0;
        int right = (numbers.size()-1);
        int check = 0;

        check = target - numbers[right];

        while(left < right){
            while(numbers[left] < check){left++;}
            if(numbers[left] == check)
            {
                ans.push_back(left + 1);
                ans.push_back(right + 1);
                return ans;
            }
            right--;
            check = target - numbers[right];
        }
        
    }
};
